src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -18,7 +18,7 @@ namespace System
     [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public partial struct Guid : IFormattable, IComparable, IComparable<Guid>, IEquatable<Guid>, ISpanFormattable
     {
-        public static readonly Guid Empty = new Guid();
+        public static readonly Guid Empty = default;
```

Comment:
💭 Why not leave the initializer off for this one?

Reply:
Because the auto-fixer didn't do that ;)

Reply:
Cool. Since this seems like a broader situation, I filed dotnet/roslyn-analyzers#3036 for it.